### DISCUSSION TITLE
Project Status 2024: No longer maintained?

### DISCUSSION THREAD

**Discussion by nwhitmont** [cheeriojs/cheerio#3754] [General]
@fb55 

The `cheeriojs/cheerio` project appears to be no longer maintained. 

It has been almost 2 years since the last release [1.0.0-rc.2](https://github.com/cheeriojs/cheerio/releases/tag/1.0.0-rc.2)

Is the Cheerio project dead or abandoned?

![cheerio-npm](https://github.com/cheeriojs/cheerio/assets/3075047/95121e12-723c-4c90-a8d1-3873e2211862)

**Comment by fb55** [comment]
Neither dead nor abandoned. The project is stable and doesn't have an urgent need for a release. Plus, I don't want to do another RC release (it has been 10+ years since the last non-RC release), and a proper release requires a bit more work than just an npm publish.

**Comment by arsekil** [comment]
Currently, it's version 1.0.0-rc.12, so no, it's definitely NOT abandoned.

**Comment by Mister-Hope** [comment]
@fb55 

Node 22 should be added in test. Also I think we should try to show a ci result (green tick) with the commit history.

> The project is stable and doesn't have an urgent need for a release

If the project is stable, then it's version number should be a stable version like `1.0.0`, I would not be curious that a project remain `1.0.0` for years, but would be if it's a non-stable version. I believe there will be more users like me have to waste time checking what's wrong with `cheerio` for rc version with latest and another time wasting checking its maintenance after some time. 

### No matter what's your attitude, it's definitely strange to remain a rc version on latest tag for a long time.

>  a proper release requires a bit more work than just an npm publish.

If so, then this is conflict with `no need for a new release`, if you have no patch fixes, code tweaks, major deps bumps or other things remaining unpublished, then this should be easy for you to release a new version to just converting it to `1.0.0`

2 years time and words like `The project is stable` already proves that the API should be regarded solid.

finally, this is strange and matters, even for open source:

> I would not be curious that a project remain `1.0.0` for years, but would be if it's a non-stable version. I believe there will be more users like me have to waste time checking what's wrong with `cheerio` for rc version with latest and another time wasting checking its maintenance after some time.
